src/theory/datatypes/theory_datatypes.h

```diff
@@ -41,22 +41,22 @@ namespace datatypes {
 
 class TheoryDatatypes : public Theory {
 private:
-  typedef context::CDChunkList<Node> NodeList;
-  typedef context::CDHashMap< Node, int, NodeHashFunction> NodeIntMap;
-  typedef context::CDHashMap< Node, bool, NodeHashFunction > BoolMap;
-  typedef context::CDHashMap< Node, Node, NodeHashFunction > NodeMap;
+ typedef context::CDList<Node> NodeList;
```

Comment:
clang-format currently expects public/private/protected to be indented by one, which is why we run into issues like this. 

Can you indent the private above and this block by one space?  That is:

class TheoryDatatypes : public Theory {
 private:
  typedef context...
  ...

I'm not sure why this is the default behavior for clang-format.

Reply:
Good catch, I fixed it.